helm/csi-powerstore/templates/controller.yaml

```diff
@@ -296,7 +296,7 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: driver
-          image: {{ required "Must provide the PowerStore driver container image." .Values.images.driver }}
+          image: {{ required "Must provide the Powerstore driver image repository." .Values.images.driverRepository }}/{{ .Chart.Name }}:{{ .Values.version }}
```

Comment:
Powerstore => PowerStore
same comment for line 94 also.

Reply:
Fixed